backend/controllers/userDetails.js

```diff
@@ -17,7 +17,22 @@ const userDetailsFactory = (
 
     const searchResultsUrl = req.session.searchResultsUrl ? req.session.searchResultsUrl : `${searchUrl}/results`
 
-    const [user, roles, groups] = await getUserRolesAndGroupsApi(res.locals, username, hasMaintainDpsUsersAdmin)
+    const [user, roles, groups, assignableGroups] = await getUserRolesAndGroupsApi(
+      res.locals,
+      username,
+      hasMaintainDpsUsersAdmin,
+    )
+    let groupsMaintainGroupManager = []
+    let groupsNotGroupManager = []
```

Comment:
would prefer `const` here, you could do the assignment by calling a separate method and then doing
```
const {groupsMaintainGroupManager, groupsNotGroupManager} = ...
```